relay.go

```diff
@@ -274,7 +274,12 @@ func (r *Relayer) canHandleNewCall() bool {
 
 func (r *Relayer) getDestination(f lazyCallReq, cs relay.CallStats) (*Connection, bool, error) {
 	if _, ok := r.outbound.Get(f.Header.ID); ok {
-		r.logger.WithFields(LogField{"id", f.Header.ID}).Warn("received duplicate callReq")
+		r.logger.WithFields(
+			LogField{"id", f.Header.ID},
+			LogField{"caller", string(f.Caller())},
+			LogField{"dest", string(f.Service())},
```

Comment:
Can we stick to caller/callee or source/dest? Mixing metaphors is a bit confusing.